.github/workflows/userbenchmark-a100.yml

```diff
@@ -24,34 +24,19 @@ jobs:
         uses: actions/checkout@v3
         with:
           path: benchmark
-      - name: Create conda environment
-        run: |
-          . "${SETUP_SCRIPT}"
-          python benchmark/utils/python_utils.py --create-conda-env "${CONDA_ENV_NAME}"
-          sudo python benchmark/utils/cuda_utils.py --setup-cuda-softlink
       - name: Tune Nvidia GPU
         run: |
           . "${SETUP_SCRIPT}"
           sudo nvidia-smi -pm 1
           sudo nvidia-smi -ac 1215,1410
           nvidia-smi
-      - name: Install PyTorch nightly
+      - name: Check PyTorch nightly if scheduled
+        if: ${{ github.event_name == 'schedule' }}
```

Comment:
Won't this just not install any nightly nor docker if the event was manual (not scheduled)? Should the step be duplicated below with an 
```
if: ${{ github.event_name != 'schedule' }}
```

Reply:
The nightly has already been installed in docker. If it is not 'schedule', we can assume the nightly has already been installed, but I can add a step below to make sure the versions are consistent.

THe installation step is at https://github.com/pytorch/benchmark/blob/main/docker/torchbench-nightly.dockerfile#L22